semgrep/tests/e2e/snapshots/test_check/test_multi_subshell_input/results.json

```diff
@@ -1,48 +1,6 @@
 {
   "errors": [],
   "results": [
-    {
```

Comment:
I think the new behavior for deduping is better

Reply:
Yeah. Though not sure why it wasn't deduping this before?